cli/common/utils.go

```diff
@@ -568,6 +572,37 @@ func ParseDependencies(rawDeps []string) (PackDependencies, error) {
 	return deps, nil
 }
 
+func getFormattedTarantoolVersion(version string, packType string) (string, error) {
```

Comment:
To be honest I'm not sure you need to parse version manually. Why don't you use go-version (https://github.com/tarantool/cartridge-cli/blob/master/go.mod#L21)?